docs/proposals/cli/README.md

```diff
@@ -0,0 +1,143 @@
+# orkactl
+
+## Requirements
+
+For the v0 of orka, only some services are available, which you can see below:
+
+### Provided services
+
+- single tenant  
+- single node  
+- single workload kind (containers)  
+
+
+### Out of scope services (for v0)
+
+- Ressource limitation  
+- Multi tenant
+- Multi workload (VM, containers, bare metal)
+- Scalabilty
+- High Availability
+- Upgradable
+- Monitoring
+- Per tenant networks
+
+
+## High Level Architecture
+
+We are the CLI. As such, we will only communicate with the Cluster API and get information from it.
+
+```mermaid
+---
+title: "Orka: High-level Architecture"
+---
+flowchart LR
+    cli[CLI]
+
+    subgraph Cluster
+        subgraph Control Plane
+            api[Cluster API]
+            ctr[Controller]
+            sch[Scheduler]
+        end
+
+        subgraph Nodes
+            subgraph Node1
+                a1[Agent]
+            end
+            subgraph Node2
+                a2[Agent]
+            end
+            subgraph Node3
+                a3[Agent]
+            end
+        end
+
+    end
+
+    cli <--> api
+    api <--> ctr
+    ctr <--> sch
+
+    a1 & a2 & a3 --> sch
+
+    a1 ~~~ a2 ~~~ a3
+
+```
+
+
+## Transport Protocol
+
+As a CLI, our component interface will communicate to the ClusterAPI using the HTTPS Protocol.
+
+Our CLI will contact the API through the following routes:
```

Comment:
Those are not routes, they're HTTP requests.

Reply:
We meant "route" as in "routing to a certain endpoint". Would you consider "HTTP Requests" to be more accurate, or shall we explain it better ?

Reply:
The former, 'HTTP requests' is a more accurate term.